### PR TITLE
images: Move fedora-testing to Fedora 33

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
* [x] image-refresh fedora-testing

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-1393-20201116-144343/

File: images/scripts/fedora-32.bootstrap [hunk 1]
```diff
@@ -18,4 +18,4 @@
 # 02110-1301 USA.
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "https://dl.fedoraproject.org/pub/fedora/linux/releases/32/Server/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "https://dl.fedoraproject.org/pub/fedora/linux/releases/33/Server/x86_64/os/"
```
Comment (martinpitt) [inline images/scripts/fedora-32.bootstrap:21]: Err, no -- you need to adjust the fedora-testing.* symlinks :grin:

Comment (allisonkarlitskaya) [reply images/scripts/fedora-32.bootstrap:21]: lol.  or read a patch before pushing it :rofl:

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-testing-20201116-150505

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-1393-20201116-150606/

**Comment by martinpitt** [comment]
You need to move the fedora-testing.install symlink as well. That does *not* require an image rebuild, so do that when you force-push this PR to include the rebuilt SHA.

**Comment by allisonkarlitskaya** [comment]
Technically we didn't even require this image rebuild, but .. well, ya.. that was a bit gross...

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-testing-20201116-152727

**Comment by allisonkarlitskaya** [comment]
Running tests in cockpit-project/cockpit#14906

**Comment by marusak** [comment]
```
# If version is not number (testing/i386) then use Fedora 32                    
if ! [ "$version" -eq "$version" ] 2>/dev/null; then version=32; fi
```
(from images/scripts/fedora.setup)

**Comment by allisonkarlitskaya** [comment]
> ```
> # If version is not number (testing/i386) then use Fedora 32                    
> if ! [ "$version" -eq "$version" ] 2>/dev/null; then version=32; fi
> ```
> 
> (from images/scripts/fedora.setup)

thanks :)

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-11.
Log: https://logs.cockpit-project.org/logs/image-refresh-1393-20201116-162424/

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpituous/bots/commits/image-refresh-fedora-testing-20201116-164949

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**